scripts/uninstall.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+
+# Uninstall libMata installed with 'make install'.
+#
+# Requires sudo privilages. Run at your own risk.
+
+sudo rm -rf /usr/local/lib/libmata.a
```

Comment:
Are you sure, that the installation is always to this folder? Would it maybe be better to have some variable set instead? I also think this will work on Linux only, or not?

Reply:
That is a great question. I was convinced MacOS has the same hierarchical structure for user-installed libraries, but now that you mention it, I do not know. I will investigate.

Reply:
To be honest, I am not even sure, that this holds for all linux Distros :). Because one can have multiple library stores.

Reply:
I think what we should do is to implement the uninstallation feature together with the refactorization of the Mata CMake Juraj is currently working on. There, we can use the CMake environment variables, which specify where to install Mata. This should be more portable and compatible with more distros as well.